monkey/infection_monkey/dropper.py

```diff
@@ -145,13 +145,9 @@ def start(self):
         if OperatingSystem.Windows == SystemInfoCollector.get_os():
             # TODO: Replace all of this string templating with a function that accepts
             #       the necessary parameters and returns a list of arguments.
-            monkey_cmdline = (
-                MONKEY_CMDLINE_WINDOWS % {"monkey_path": self._config["destination_path"]}
-                + monkey_options
-            )
-            monkey_cmdline_split = shlex.split(
-                monkey_cmdline,
-                posix=False,  # won't try resolving "\" in paths as part of escape sequences
+
+            monkey_cmdline, monkey_cmdline_split = get_monkey_cmd_lines_windows(
+                MONKEY_CMDLINE_WINDOWS, self._config["destination_path"], monkey_options
```

Comment:
Why do we need to pass `MONKEY_CMDLINE_WINDOWS` as an argument? Where else would we have to use `get_monkey_cmd_lines_windows()` with a different first argument (cc: @mssalvatore)? If nowhere, let's import `MONKEY_CMDLINE_WINDOWS` in `commands.py` and remove the first parameter of `get_monkey_cmd_lines_windows()`.